elastic/kibana
# [Console Monaco Migration] Add autosave to localStorage

Closes https://github.com/elastic/kibana/issues/180211

## Summary

This PR adds support for autosaving the current Console editor value to local storage.

**How to test:**
1. Create a `config/kibana.dev.yml` file (if one doesn't exist already) and add the line: `console.dev.enableMonaco: true`
2. Start Es and Kibana
3. Go to Dev Tools -> Console
4. Type in some input in Console
5. Navigate to another page and come back to Console. Verify that the input is the same as the one you last typed.
